README.md

```diff
@@ -35,4 +37,7 @@ A unity platforming game, featuring ugprades and differing levels. Made by Ediso
   - This is simply stored as a JSON format.
 - There are two scenes, MainMenu and SampleScene, where the MainMenu is the beginning menu, and SampleScene is the main game itself. 
 ### Level Generator
+- imageGenerator.py is the Python file used to generate the levels
+  - Note: In order to run this file, you will need to have the PILLOW library installed in Python. Installation instructions can be found here: https://pillow.readthedocs.io/en/stable/installation.html
```

Comment:
maybe do the link thing 
`[PILLOW](https://pillow.readthedocs.io/en/stable/installation.html)`